app/src/main/res/values/themes.xml

```diff
@@ -12,14 +12,28 @@
         <!-- Status bar color. -->
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
+        <item name="buttonStyle">@style/Theme.NCNF.BlueButton</item>
+        <item name="android:editTextColor">@color/light_cyan</item>
+        <item name="android:textColorHint">@color/light_cyan</item>
```

Comment:
One thing I forgot: Can you change these two back to black, since the gradient has been changed from a dark one to a light one?

Reply:
Yep this is done